### ISSUE TITLE
Latex Error in longtabu Environment

### ISSUE THREAD

**Issue by Christian-fhs** [doxygen/doxygen#6837]
Hello,

first I want to describe my Problem.
- I am running Doxygen 1.8.15 and i updated my Miktex Libraries. The concerning error occcures in the tabu environment, which I use in version 2.10.
- I created a small test project which consists of the generated header.tex, footer.tex and doxygen.sty. The only documented file is test.h with the following content:
```
// *****************************************************************************
/// @file   test.h
///
/// First Header  | Second Header
/// ------------- | -------------
/// Content Cell  | Content Cell
/// Content Cell  | Content Cell
// *****************************************************************************
```
This example should create the tabular above (which is copied from the doxygen manual page). The Latex compile process is interrupt by the following message:

> ! Missing } inserted.
> <inserted text> 
>                 }
> l.18 \end{longtabu}

The regarding file's content is the following:

```
\hypertarget{test_8h}{}\section{test.\+h-\/Dateireferenz}
\label{test_8h}\index{test.h@{test.h}}


\subsection{Ausführliche Beschreibung}
\tabulinesep=1mm
\begin{longtabu}spread 0pt [c]{*{2}{|X[-1]}|}
\hline
\PBS\centering \cellcolor{\tableheadbgcolor}\textbf{ First Header  }&\PBS\centering \cellcolor{\tableheadbgcolor}\textbf{ Second Header   }\\\cline{1-2}
\endfirsthead
\hline
\endfoot
\hline
\PBS\centering \cellcolor{\tableheadbgcolor}\textbf{ First Header  }&\PBS\centering \cellcolor{\tableheadbgcolor}\textbf{ Second Header   }\\\cline{1-2}
\endhead
Content Cell  &Content Cell   \\\cline{1-2}
Content Cell  &Content Cell   \\\cline{1-2}
\end{longtabu}
```

If more information is necessary I will provide it.
This error is driving me crazy. I am using the latest libraries and a current version of doxygen, still this simple tabular can not be compiled into a pdf.
I may have to add, that I am no doxygen poweruser and the error could be a result of my unaware usage of doxygen.

**Comment by albert-github** [comment]
Thanks for the case, but I think the best is to report the problem also to:
- https://github.com/tabu-fixed/tabu/issues/1
or
- https://github.com/tabu-fixed/tabu/issues/new

as the problem is due to the change in  latex3/latex2e (https://github.com/latex3/latex2e) which unfortunately broke the (unmaintained) tabu package.

**Comment by Christian-fhs** [comment]
What could be the quickest way to get it running again? 
Is it possible to downgrade the libraries? 
What would you suggest?

**Comment by albert-github** [comment]
To get it running again
- report the bug at the places indicated
- yes downgrad to a LateX version from before mid of December 2018 (i.e. before the time the fix was made in latex3/latex2e)

**Comment by Christian-fhs** [comment]
Thank you for your fast help, I appreciate that.
What would be the best way to reverse the Latex3 release to `2018-04-01`, so that I could use doxygen again?
Am I assuming right, that Latex3 is distributed through CTAN, which does not track old releases?

**Comment by albert-github** [comment]
I never did it myself so it is a bit hard to advise and also depending on the distribution you use (TexLive, MikTeX, MacTeX ...), but I think in one way or another you need to get an old distribution.
Best to google for an old distribution or check https://tex.stackexchange.com

**Comment by dmcclimans** [comment]
This continues to be a problem, and it doesn't look like the tabu errors are going to be fixed. The https://github.com/tabu-fixed/tabu page says it is still broken, they don't intend to do more fixes, and they do not recommend its use. 

This means that pdf output is broken in doxygen. I'm not a latex expert, but using MikTex on Windows it seems to be impossible to keep it from updating, so you end up with the broken version whether you want it or not.

Is it possible to change doxygen so that it does not rely on the tabu package?

**Comment by veelo** [comment]
> Is it possible to change doxygen so that it does not rely on the tabu package?

That seems like the only good solution to me. Doxygen switched to tabu in version 1.8.11. Unfortunately for me, I cannot downgrade to that as I rely on a fix that was done in 1.8.12 (Fix uppercase letters B-Z being unnecessarily escaped in index HTML anchors).

I hope the Doxygen team will be able to revert [the tabu change](https://github.com/doxygen/doxygen/commit/647b6ac8669cd8ba1e8c60eeb3c2de961c7d6a1b) as soon as possible.

**Comment by veelo** [comment]
This looks to be a duplicate of #6769.

**Comment by veelo** [comment]
I think it is safe to close this, as the problem has been fixed in doxygen version 1.8.16.

**Comment by albert-github** [comment]
Reference from @veelo is correct, so the problem has been solved in the mean time and setting the appropriate label.
